src/styles/index.scss

```diff
@@ -1,3 +1,142 @@
-body {
+@import "stars";
+
+*{
+  box-sizing: border-box;
+}
+
+html {
+  font-family: Roboto, sans-serif;
+  font-style: normal;
+  font-weight: 400;
+}
+
+body,
+h1,
+p{
   margin: 0;
+
+}
+
+.header{
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: center;
+  align-items: center;
+  background-color: #FFFF;
+
+
+}
+
+.container{
+ width: 200px;
+ padding: 32px 16px 16px;
+ border-radius: 5px;
+ border: 1px solid #f3f3f3;
+}
+
+.container__image{
+  display: flex;
+  justify-content: center;
+  margin-bottom: 40px;
+
+}
+
+.image{
+  width: 160px;
+  height: 134px;
+}
+
+.container__name{
+  color:  #060B35;
+  font-family: Roboto, sans-serif;
+  font-size: 12px;
+  font-style: normal;
+  font-weight: 500;
+  line-height: 18px;
+  margin-bottom: 4px;
+
+
+}
+
+.container__product-code{
+  display: flex;
+  justify-content: flex-start;
+  margin-bottom: 16px;
+  font-family: Roboto, sans-serif;
```

Comment:
Consider applying the font family for the entire page to get it inherited by elements